script.artistslideshow/addon.xml

```diff
@@ -10,6 +10,8 @@
 		<provides>executable</provides>
 	</extension>
 	<extension point="xbmc.addon.metadata">
+		<news>2.0.3
+		- removed support for htbackdrops.org (API access no longer avaialble)</news>
```

Comment:
You could update the PR by force-pushing.
If it is too much hassle for you then I would also be fine to just keep that typo included. What´s your preference?